app/src/main/res/layout-land/fragment_details.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Same comment as the main fragment.

Reply:
@rob729 , ok I'll try it out, if it works fine I will remove the landscape file.